src/cartesi-machine.lua

```diff
@@ -257,8 +257,80 @@ where options are:
   --auto-reset-uarch-state
     automatically reset state after miroarchitecture halt
 
-  -i or --htif-console-getchar
-    run in interactive mode.
+  --htif-console-getchar
+    run in interactive mode
+
+  --virtio-console
+    add a VirtIO console device,
+    this console is more responsive than HTIF console and has support for terminal size
+
+  -i
+    run in an interactive mode using a VirtIO console device
+    same as --htif-console-getchar --virtio-console
+
+  --share-dir=<directory>
```

Comment:
Here I would try to use something as similar to Docker as possible. Perhaps even a -v/--volume options. Just so people don't have to learn something new. This means that I would always automount using the specified target path.

Reply:
You could add a --virtio-9p option that only sets the tag and the host-directory and does not do automounting.